Project-1/TimPrudhomme_Project1.md

```diff
@@ -0,0 +1,483 @@
+3. Let V be the vocabulary of first-order logic consisting of a binary relation P and a unary relation F. Interpret P(x,y) as “x is a parent of y” and F(x) as “x is female.” Where possible define the following formulas in this vocabulary; where not possible, explain why: 
+```
+  (a)  B(x,y) that says that x is a brother of y
+  (b)  A(x,y) that says that x is an aunt of y
+  (c)  C(x,y) that says that x and y are cousins 
+  (d)  O(x) that says that x is an only child
+  (e)  T(x) that says that x has exactly two brothers 
+```
+
+## Answers:
+```
+FOL 
+
+// x is brother of y
+B(x,y) = ∃x∃y∃p( P(p,x) ∧ P(p,y) ∧ ¬F(x) ∧ x≠y ∧ x≠p ∧ y≠p )
+
+// x is aunt of y
+A(x,y) = ∃y∃p∃g∃x( P(p,y) ∧ P(g,p) ∧ P(g,x) ∧ F(x) ∧ x≠y ∧ x≠p ∧ x≠g ∧ p≠g ∧ p≠y ∧ g≠y )
+
+// x is cousin of y
+C(x,y) = ∃x∃p∃g∃u∃y( P(p,x) ∧ P(g,p) ∧ P(g,u) ∧ P(u,y) ∧ x≠y ∧ x≠p ∧ x≠g ∧ x≠u ∧ p≠g ∧ p≠y ∧ p≠u ∧ g≠y ∧ g≠u) )
+
+// x is an only child
+O(x) = ∃x∃p( P(p,x) ∧ x≠p ∧ ∀y( P(p,y) → y=x ∧ y≠p) )
+
+// x has exactly two brothers
+T(x) = ∃x∃y∃w∃p(
+                  P(p, x) ∧ x≠p
+                  ∧ P(p,y) ∧ y≠p ∧ ¬F(y)
+                  ∧ P(p,w) ∧ w≠p ∧ ¬F(w) 
+                  ∧ ∀b( P(p,b) → (b=y ∨ b=w) ∧ b≠p )
+                )
+```
+
+---
+# Question 4
+4. Let V be a vocabulary of the attribute (concept) language with complements (ALC) consisting of a role name "parent_of" and a concept name "Male". Interpret parent_of as "x is a parent of y" and M as "x is male". Where possible define the following formulas in this vocabulary; where not possible, explain why: 
+```
+  (a)  B that says that x is a brother of y
+  (b)  A that says that x is an aunt of y
+  (c)  C that says that x and y are cousins 
+  (d)  O that says that x is an only child
+  (e)  T that says that x has exactly two brothers 
+```
+
+## Answers:
+### $ALC$
+* The description logic $AL$ conventionally consists of:
+  * Top $\top$, Bottom $\bot$, Conjunction $\sqcap$, Atomic Negation $\neg A$, Value Restriction $\forall r.C$
+* $AL$ extended with $C$ adds Negation (of compound concepts) $\neg C$
+  * This allows a derivation of Disjunction $\sqcup$ (from Negation & Conjunction)
+  * This allows a derivation of Existential Restriction $∃r.C$ (from Negation & Value Restrction)
+
+### DL Extensions
+* None of the provided formulas are strictly definable as roles in $ALC$ in terms of the role `parent_of`
+* The DL extension $H$ adds Role Inclusion Axioms $r \sqsubseteq s$, allowing us to define other roles in terms of `parent_of`
+* The DL extension $I$ adds Inverse Roles $r^-$, allowing us to express the inverse of `parent_of` 
+
+With these extensions we can define the formulas:
+
+(a) `B that says that x is a brother of y`  
+$B ≡ M\sqcap ∃parentOf^-.(∃parentOf.\top)$  
+
+  * "Someone is a brother of someone else iff they're a male and there's a parent of that someone who is also a parent of that someone else"
+  * `TODO:` This relation should also be defined as irreflexive, symmetric, and transitive
```

Comment:
Good stuff, with the exception that 'brother of' shouldn't be symmetric. John is brother of Kellye shouldn't entail Kellye is brother of John.